gensrc/proto/lake_types.proto

```diff
@@ -86,6 +86,15 @@ message TabletMetadataPB {
     optional PersistentIndexTypePB persistent_index_type = 12;
 }
 
+enum TabletMetaTypePB {
+    ENABLE_PERSISTENT_INDEX = 0;
+}
+
+message TabletMetaModifyPB {
+    required TabletMetaTypePB tablet_meta_type = 1;
```

Comment:
Instead of using a field to identify the target metadata to be operated on, I prefer to add a field to identify the type of operation: UPDATE or DELETE. As for the target metadata to be modified, you can use protobuf's has_xxx() to determine which fields exist, and the ones that do exist are the ones that are to be modified.